### PR TITLE
A few descriptions.

### PR DESCRIPTION
**Pull request by Tarlan2**
-corrected the descriptions for the SMG crate and the bonesaw.
-added a few ones for items that had none.

### CODE REVIEW

**Review by SweptWasTaken — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Looks k

**Comment by UnderscoreX5** [comment]
Really like them!

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@
   name: x-ray cannon
   parent: BaseItem
   id: XrayCannon
-  description: Blah blah blah...
+  description: AKA the Cancer Cannon.
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:83]: Couldn't get something more creative?

**Review by YakovLya — APPROVED** [review]
Good job

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**